### ISSUE TITLE
Documentation bug: Incorrect example in model.tex

### ISSUE THREAD

**Issue by rdipietro** [blei-lab/edward#594]
Snippet from `http://edwardlib.org/api/model`, under `Examples`:

```
z1 = tf.constant([[2.0, 8.0]])
z2 = tf.constant([[1.0, 2.0]])
x = Bernoulli(p=tf.matmul(z1, z2))
```

This `matmul` doesn't make sense because we're trying to multiply two row vectors.

I'd try to fix it but don't know what's intended. (I suppose we could transpose `z2` to be a column vector, but then we'd still have `p > 1`.)

**Comment by dustinvtran** [comment]
This is fixed in master (https://github.com/blei-lab/edward/blob/master/edward/models/random_variable.py#L67), although not pushed to the website.

The example is meant to represent a Bernoulli matrix factorization, where we are modeling a 2 x 2 matrix of 0s and 1s. (That said, the outputted matrix mul should have values between 0 and 1, which is still not true in master. So maybe we should maybe make it parameterize `logits` instead of `p` of Bernoulli.)

**Comment by rdipietro** [comment]
How about

```
z1 = tf.constant([[1.0, -0.8], [0.3, -1.0]])
z2 = tf.constant([[0.9, 0.2], [2.0, -0.1]])
p = tf.sigmoid(tf.matmul(z1, z2))
x = Bernoulli(p=p)
```

If that's fine I'll replace in both spots.

**Comment by rdipietro** [comment]
Ah never mind – just noticed that TF RVs accept logits directly. I'll submit a PR with

```
z1 = tf.constant([[1.0, -0.8], [0.3, -1.0]])
z2 = tf.constant([[0.9, 0.2], [2.0, -0.1]])
logits = tf.matmul(z1, z2)
x = Bernoulli(logits=logits)
```

**Comment by rdipietro** [comment]
Taken care of with https://github.com/blei-lab/edward/pull/595
